cypress/integration/26-register_registration/creator_navigates_to_resource_type_tab/test_subtypes.js

```diff
@@ -0,0 +1,19 @@
+import { Given, When, Then, And } from 'cypress-cucumber-preprocessor/steps';
+
+Then('they see a list of subtypes:', (dataTable) => {
+  cy.get('@subTypes').then((subTypes) => {
+    if (Object.keys(subTypes).length > 0) {
+      cy.get('[data-testid=publication-instance-type]').type(' ').click({ force: true });
+      cy.testDataTestidList(dataTable, subTypes);
+    }
+  });
+});
+
+And('they select the Subtype {string}:', (subType) => {
+  cy.get('@subTypes').then((subTypes) => {
+    if (Object.keys(subTypes).length > 0) {

```

Comment:
burde ikke sjekken her egentlig være `if (subTypes[subType])`? det er den verdien som brukes senere, så det kan godt være at dette objektet har verdier, men ikke den vi faktisk vil ha. 

kan være jeg som tenker feil også, og at slik det er nå fører til en failed test og avdekker dermed feil i appen

Reply:
Dette er for å ungå at vi får failed test på ting som ikke er implementert siden jeg har satt subTypes til et tomt objekt i de tilfellene. Syns det gir bedre kontroll.